assets/constants.js

```diff
@@ -5,4 +5,5 @@ const PUB_SUB_EVENTS = {
   quantityUpdate: 'quantity-update',
```

Comment:
Okay, I expected this question :). I thought that it makes sense to add the spinner only for cases when we request updated data through API because that is the case when we potentially may have a delay. If you try to add to cart or using cart-drawer change qty in cart you will see the spinner.

Does it make sense?